mail_composer_cc_bcc/tests/test_mail_cc_bcc.py

```diff
@@ -32,38 +26,16 @@ def open_mail_composer_form(self):
         ctx = {
             "default_partner_ids": test_record.ids,
             "default_model": test_record._name,
-            "default_res_id": test_record.id,
+            "default_res_ids": test_record.ids,
             # to ensure consistent test results even when mail_post_defer is installed
             "mail_notify_force_send": True,
         }
         form = Form(self.env["mail.compose.message"].with_context(**ctx))
         form.body = "<p>Hello</p>"
         return form
 
-    def open_invoice_mail_composer_form(self):
-        # Use form to populate data
-        for_name = [("name", "like", "%INV/20__/00003")]
-        self.test_invoice = test_record = self.env["account.move"].search(for_name)
-        self.assertTrue(
-            self.test_invoice,
-            "Test setup did not succeeed. Invoice not found.",
-        )
-        ctx = {
-            "active_ids": test_record.ids,
-            "default_model": "account.move",
-            "default_res_id": test_record.id,
-            "mail_notify_force_send": True,
-        }
-        form = Form(self.env["account.invoice.send"].with_context(**ctx))
-        form.body = "<p>Hello</p>"
-        return form
-
-    def test_upstream_file_hash(self):
-        """Test that copied upstream function hasn't received fixes"""
-        func = inspect.getsource(upstream._send).encode()
-        func_hash = hashlib.md5(func).hexdigest()
-        self.assertIn(func_hash, VALID_HASHES)
```

Comment:
can you keep this test? (and update hash)
this is a mechanism to let us know when upstream method got changed as we need to keep track of eventual important changes that we should update in our override as well

see `https://github.com/OCA/social/pull/1223` for example